apps/frontend/src/components/fap/Proposals/FapProposalsAndAssignmentsTable.tsx

```diff
@@ -765,6 +765,15 @@ const FapProposalsAndAssignmentsTable = ({
           options={{
             search: true,
             selection: true,
+            pageSize: Math.min(10, FapProposalsWitIdAndFormattedDate.length),
+            pageSizeOptions: [
```

Comment:
This looks good to me but you can maybe just put the array in a variable called `pageSizeOptions` or something similar and then just use that one here instead of doing the calculation and filtering directly in the html code.